packages/mask/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {},
   "dependencies": {
     "@balancer-labs/sor": "^1.0.0",
+    "@cyberlab/cyberconnect": "^4.2.2",
```

Comment:
Remove this since it is installed in `packages/plugins/CyberConnect/package.json`